decidim-proposals/app/controllers/decidim/proposals/proposals_controller.rb

```diff
@@ -225,23 +225,16 @@ def search_collection
       def default_filter_params
         {
           search_text_cont: "",
-          with_any_origin: default_filter_origin_params,
+          with_any_origin: nil,
           activity: "all",
-          with_any_category: default_filter_category_params,
+          with_any_category: nil,
```

Comment:
check the comment on decidim-budgets/app/controllers/decidim/budgets/projects_controller.rb